packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2654,30 +2654,26 @@ export class ContainerRuntime
 				return;
 			}
 
-			const messages: {
-				message: InboundSequencedContainerRuntimeMessage;
-				localOpMetadata: unknown;
-			}[] = local
-				? this.pendingStateManager.processPendingLocalBatch(
-						inboundBatch.messages,
-						inboundBatch.batchStartCsn,
-						inboundBatch.emptyBatchSequenceNumber,
-					)
-				: inboundBatch.messages.map((message) => ({ message, localOpMetadata: undefined }));
-			if (messages.length === 0) {
+			// Reach out to PendingStateManager, either to zip localOpMetadata into the *local* message list,
+			// or to check to ensure the *remote* messages don't match the batchId of a pending local batch.
```

Comment:
Oops this line is from the future, I'll fix the comment for now.  (I pulled this out of another WIP branch)